rs/tests/consensus/tecdsa/BUILD.bazel

```diff
@@ -62,6 +65,7 @@ system_test_nns(
     proc_macro_deps = MACRO_DEPENDENCIES,
     tags = [
         "k8s",
+        "slow_test",  # since it takes longer than 5 minutes.
```

Comment:
Does this test also need `extra_head_nns_tags = []` like the others?

Reply:
Yes, thanks for spotting it. I added it.